### PR TITLE
#45: add error message and fix error with not existing class

### PR DESCRIPTION
**Pull request by mmezhensky**
### Description (*)

Fixed error with existing class in di.xml.
Added error message if not an existing class.

### Fixed Issues (if relevant)

1. magento/magento2-phpstorm-plugin#45: Action. Plugin method generation NullPointerException issue

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: resources/magento2/validation.properties [hunk 1]
```diff
@@ -10,3 +10,4 @@ validator.file.alreadyExists={0} already exists
 validator.file.cantBeCreated={0} can't be created
 validator.class.alreadyDeclared={0} already declared in the target module
 validator.magentoVersionInvalid=Please specify valid Magento version or use 'any' keyword as default
+validator.magentoNotExistTargetClass=Target class is not found. Check the di.xml file
```
Comment (eduard13) [inline resources/magento2/validation.properties:13]: As far as this validation message is related to class, please add it in `validator.class` scope
```suggestion
validator.class.targetClassNotFound=Target class is not found. Check the di.xml file
```

Comment (eduard13) [reply resources/magento2/validation.properties:13]: Hi @mmezhensky could you please adjust the name, and also adjust it where it is used?
Thank you.

Comment (mmezhensky) [reply resources/magento2/validation.properties]: done, please check. Tnx!

**Comment by VitaliyBoyko** [comment]
@eduard13 @mmezhensky  Thank you, guys!

**Pull request merged**